osu.Game.Tests/Editing/TestSceneHitObjectComposerDistanceSnapping.cs

```diff
@@ -61,6 +61,7 @@ public TestSceneHitObjectComposerDistanceSnapping()
             BeatDivisor.Value = 1;
 
             composer.EditorBeatmap.Difficulty.SliderMultiplier = 1;
+            composer.EditorBeatmap.BeatmapInfo.DistanceSpacing = 1;
```

Comment:
Incidentally, why isn't `BeatmapInfo.DistanceSpacing` initialised by default to 1 in the model class itself? Seems like a better idea than patching readers all over.

Reply:
I'm curious if its the default value and not because of the test beatmap. Will see.

Reply:
There are three cases of this currently:
 - [`TestSceneObjectObjectSnap`](https://github.com/ppy/osu/pull/16576/files#diff-ba9e43f7e802590bea73bf4272d1f8318c3e94b868927ec612d0cb3485280c40) and [`TestSceneSliderSnapping`](https://github.com/ppy/osu/pull/16576/files#diff-fca3d8bdcf3b7b013aafcb562ebb2743e1d9a25029f5334401e38a563de58dd5), which awaits decision at https://github.com/ppy/osu/pull/16576#discussion_r860450100. 
 - [`TestSceneHitObjectComposerDistanceSnapping`](https://github.com/ppy/osu/pull/16576/files#diff-2d74227b54f85dd4814534d6da5bfc90cc1a69f57ad9cc5ba1b1fe14b2db948a), which has been intentionally left inline with the pre-existing `BeatDivisor = 1` and `SliderMultiplier = 1`, unless we want to mutate the values in `TestBeatmap` itself which I'm not 100% sure about (would basically break how the beatmap was mapped).

Reply:
I think `1` should be the default value in all cases. `0` doesn't make any sense.

Reply:
Misunderstood this. It seems that the actual test beatmap ("My Love") provides a non-default value. In which case I think this is fine (maybe include an inline comment noting this).